HTTP_HEADER_FORMAT.md

```diff
@@ -0,0 +1,86 @@
+# Trace Context HTTP Header Format
+
+A trace context header is used to pass trace context information across systems
+for a HTTP request. Our goal is to share this with the community so that various
+tracing and diagnostics products can operate together, and so that services can
+pass context through them, even if they're not being traced (useful for load
+balancers, etc.)
+
+# Format
+
+## Header name
+
+`Trace-Context`
+
+## Field value
+
+`base16(<version>)-<version_format>)`
+
+The value will be US-ASCII encoded (which is UTF-8 compliant). Character `-` is
+used as a delimiter between fields.
+
+### Version
+
+Is a 1-byte representing a 8-bit unsigned integer. Version 255 reserved.
```

Comment:
@adriancole I really like an explicit versioning. I think it will help a lot long term

@bogdandrutu was the idea of the version an incremental thing or the format of the header? Let's say `00` is what explained here, `01` - binary format, `02` - alternative length of request ID or other implementation? We've been thinking suggest to use it to indicate the alternative way of generating identifiers.